config/config.go

```diff
@@ -289,8 +289,8 @@ func (pconf *PluginConfig) buildCheckPlugin(name string) (*CheckPlugin, error) {
 	plugin := CheckPlugin{
 		Command:               *cmd,
 		CustomIdentifier:      pconf.CustomIdentifier,
-		NotificationInterval:  pconf.NotificationInterval,
-		CheckInterval:         pconf.CheckInterval,
+		NotificationInterval:  pconf.NotificationInterval.toMinutes(),
+		CheckInterval:         pconf.CheckInterval.toMinutes(),
```

Comment:
I'm wondering we should change CheckInterval to time.Duration or not. But maybe we can do this in later pull request (if we want).

- Change `type duration int32` to `type duration time.Duration`
- Change `CheckInterval *int32` to `CheckInterval time.Duration`
- Make https://github.com/mackerelio/mackerel-agent/blob/fe40e3c5a506ce552657789e3a32259878aaf790/checks/checker.go#L92 simple.